src/Controls/src/Core/Platform/Android/TabbedPageManager.cs

```diff
@@ -526,7 +526,7 @@ protected virtual ColorStateList GetItemTextColorStates()
 				return _newTabTextColors;
 
 			int checkedColor;
-			int defaultColor;
+			int defaultColor = 0;
```

Comment:
Is there a better way to get the Default color from the OS ?

Reply:
@jsuarezruiz please we need a comment on this.

Reply:
Right, I have added small changes and a comment.